models/points.go

```diff
@@ -1421,17 +1425,14 @@ func parseNumber(val []byte) (interface{}, error) {
 }
 
 func newFieldsFromBinary(buf []byte) Fields {
-	fields := Fields{}
+	fields := make(Fields, 5)
```

Comment:
is `5` a magic number or just something better than not allocating anything?  `5` will get you a cap of `8`, and `9` will get you a cap of `16`, but these are all implementation details of the runtime and could change from version to version (but unlikely I suspect).